Makefile.am

```diff
@@ -42,16 +43,15 @@ distwin64-dir :
 	rm -rf liblouis-$(VERSION)-win64/*
 	@if which x86_64-w64-mingw32-gcc >/dev/null; then \
 	  ./configure --host x86_64-w64-mingw32 --prefix=$(CURDIR)/liblouis-$(VERSION)-win64 && \
```

Comment:
Don't you need to set the `CPPFLAGS` and the `LDFLAGS` before invoking configure, just as you did for the `distwin32-dir` target?